java/odsa-spring/script/ODSASpringBoot.sql

```diff
@@ -0,0 +1,33 @@
+CREATE USER ODSA_USER IDENTIFIED BY <YOUR_PASSWORD>;
+GRANT DWROLE TO ODSA_USER;
+GRANT CREATE SESSION TO ODSA_USER;
```

Comment:
This is redundant, as it is already provided by the `DWROLE`, see the [documentation](https://docs.oracle.com/en/cloud/paas/autonomous-database/adbsa/manage-users-privileges.html#GUID-50450FAD-9769-4CF7-B0D1-EC14B465B873).

Reply:
This is fixed as well, thanks again for your support and the orientation/documentation.